src/server/system_services/pool_server.js

```diff
@@ -1216,6 +1217,30 @@ async function update_account_default_resource() {
     }
 }
 
+function update_issues_report(req) {
+    const { namespace_resource_id, error_code, time} = req.rpc_params;
+
+    const invalid_bucket_errors = ['ContainerNotFound', 'NotSignedUp', 'NoSuchBucket', 'InvalidSecurity',
+    'InvalidBucketState', 'InternalError', 'AccessDenied', 'InvalidAccessKeyId', 'AccountProblem'];
+
+    if (!_.includes(invalid_bucket_errors, error_code)) {
+        return;
+    }
+    const updates = {
+            _id: parse_object_id(namespace_resource_id),
+            $push: {
```

Comment:
Need to check if PostgreSQL supports this.

Reply:
@romayalon It's better to modify the array using js and not mongo operators. similar to what's done in `nodes_monitor`
https://github.com/noobaa/noobaa-core/blob/4e026d9e52822f03aab31acc9e35ed6b0ee68607/src/server/node_services/nodes_monitor.js#L3497-L3511

Reply:
np, changed to modification on js and not using mongo